javascript/javascript2/week3/warmup/warmup.js

```diff
@@ -0,0 +1,92 @@
+// 1.
+setTimeout(() => {
+  alert("Called after 2.5 seconds");
+}, 2500);
+
+// 2.
+const getDelay = (delay, stringToLog) => {
+  setTimeout(() => {
+    alert(`${stringToLog} logged after ${delay}`);
+  }, delay * 1000);
+};
+
+// Create a button in html, use the function
+let btnClickMe = document.getElementById("delayerBTN");
+btnClickMe.addEventListener("click", () => getDelay(5, "Hej Hallo Welcome"));
```

Comment:
It is not an error, but I would advise to name variable according to id name.
If the id is delayedBTN, then `let delayedBTN = document.getElementById("delayerBTN");`
I believe it it easier to follow for what reason was created this button.